.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 
 /src/libraries/Common/src/System/Net/Http/aspnetcore/               @dotnet/http
 /src/libraries/Common/tests/Tests/System/Net/aspnetcore/            @dotnet/http
-/src/libraries/System.Text.Json/                                    @ahsonkhan @steveharter @layomia @Jozkee
+/src/libraries/System.Text.Json/                                    @ahsonkhan @steveharter @layomia @Jozkee @eiriktsarpalis
```

Comment:
Is @ahsonkhan still valid here?

Reply:
That presumably is up to @ahsonkhan, since all it does is set him as a reviewer. Any reviews he does are greatly appreciated, but not expected...

Reply:
I am happy to get tagged for reviews and help from time to time, which is why I left the code owners as is :)

If there is a pressing reason to update the list, that would be OK with me as well.

Reply:
Thanks. Will leave @ahsonkhan as a code owner - we can update in the future if needed.

Reply:
Thanks @ahsonkhan!